pkg/command/util/utils.go

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2022 Singularity Data
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package util
+
+import (
+	"fmt"
+	"os"
+)
+
+// TODO: do some check for error
+
+func CheckErr(err error) {
```

Comment:
Should `PrintErr` be more precise?

Reply:
So far, it just prints the error, but we may add some judgment of the error here later, not just printing.

Reply:
Alright, I see.